platform-configs/optiplex-7010.robot

```diff
@@ -0,0 +1,217 @@
+
+${DMIDECODE_VENDOR}=                            3mdeb
+${DMIDECODE_FAMILY}=                            N/A
+${DMIDECODE_TYPE}=                              Desktop
+
+${DEVICE_USB_KEYBOARD}=                         SiGma Micro Keyboard TRACER Gamma Ivory
+${DEVICE_NVME_DISK}=                            ${EMPTY}
+${USB_MODEL}=                                   Kingston
+${USB_DEVICE}=                                  Multifunction Composite Gadget
+
+# Supported test environments
+${TESTS_IN_FIRMWARE_SUPPORT}=                   ${TRUE}
+${TESTS_IN_UBUNTU_SUPPORT}=                     ${TRUE}
+${TESTS_IN_WINDOWS_SUPPORT}=                    ${FALSE}
+
+# Regression test flags
+${DASHARO_SECURITY_MENU_SUPPORT}=               ${TRUE}
+${DASHARO_USB_MENU_SUPPORT}=                    ${TRUE}
+${DASHARO_NETWORKING_MENU_SUPPORT}=             ${TRUE}
+${DASHARO_INTEL_ME_MENU_SUPPORT}=               ${TRUE}
+${DASHARO_CHIPSET_MENU_SUPPORT}=                ${TRUE}
+${DASHARO_POWER_MGMT_MENU_SUPPORT}=             ${FALSE}
+${DASHARO_PCI_PCIE_MENU_SUPPORT}=               ${TRUE}
+${DASHARO_PCIE_REBAR_SUPPORT}=                  ${TRUE}
+${DASHARO_MEMORY_MENU_SUPPORT}=                 ${TRUE}
+# Test module: dasharo-compatibility
+${CUSTOM_BOOT_MENU_KEY_SUPPORT}=                ${TRUE}
+${CUSTOM_SETUP_MENU_KEY_SUPPORT}=               ${TRUE}
+${EXTERNAL_HDMI_DISPLAY_SUPPORT}=               ${TRUE}
+${EXTERNAL_DISPLAY_PORT_SUPPORT}=               ${TRUE}
+${CUSTOM_LOGO_SUPPORT}=                         ${TRUE}
+${USB_DISKS_DETECTION_SUPPORT}=                 ${TRUE}
+${USB_KEYBOARD_DETECTION_SUPPORT}=              ${TRUE}
+${UEFI_SHELL_SUPPORT}=                          ${TRUE}
+${UEFI_COMPATIBLE_INTERFACE_SUPPORT}=           ${TRUE}
+${IPXE_BOOT_SUPPORT}=                           ${TRUE}
+${NVME_DISK_SUPPORT}=                           ${TRUE}
+${SD_CARD_READER_SUPPORT}=                      ${TRUE}
+${AUDIO_SUBSYSTEM_SUPPORT}=                     ${TRUE}
+${EXTERNAL_HEADSET_SUPPORT}=                    ${TRUE}
+${SUSPEND_AND_RESUME_SUPPORT}=                  ${TRUE}
+${FIRMWARE_NUMBER_VERIFICATION}=                ${TRUE}
+${PRODUCT_NAME_VERIFICATION}=                   ${TRUE}
+${RELEASE_DATE_VERIFICATION}=                   ${TRUE}
+${MANUFACTURER_VERIFICATION}=                   ${TRUE}
+${VENDOR_VERIFICATION}=                         ${TRUE}
+${TYPE_VERIFICATION}=                           ${TRUE}
+${DTS_SUPPORT}=                                 ${TRUE}
+${UPLOAD_ON_USB_SUPPORT}=                       ${TRUE}
+${CPU_TESTS_SUPPORT}=                           ${TRUE}
+${L2_CACHE_SUPPORT}=                            ${TRUE}
+${L3_CACHE_SUPPORT}=                            ${TRUE}
+${MEMORY_PROFILE_SUPPORT}=                      ${TRUE}
+${ESP_SCANNING_SUPPORT}=                        ${FALSE}
+${DCU_UUID_SUPPORT}=                            ${TRUE}
+${DCU_SERIAL_SUPPORT}=                          ${TRUE}
+
```

Comment:
Do those values differ from default? If no then there is no need to include them in those files.
Same goes for 9010.
It is very hard to extract specific data for this machine if all of the variables are included.

Reply:
Right, removed all the overlap in [this commit](https://github.com/Dasharo/open-source-firmware-validation/pull/424/commits/b65bed093f6a66656720c48b4da15e841c026b89)